spec/support/features/checkouts_helper.rb

```diff
@@ -0,0 +1,13 @@
+module Features
+  def have_checked_out_book(title)
+    have_css('ul.books.unavailable li', text: title)
+  end
+
+  def have_returned_book(title)
+    have_css('ul.books.available li', text: title)
+  end
+
+  def have_available_book(title)
```

Comment:
def `have_available_book` and def `have_returned_book` do the same thing. However, the context of the test makes one more readable than the other. Is it okay to have two functions that do the same thing if readability is improved?